src/YawPitchControl/utils.js

```diff
@@ -15,35 +15,25 @@ export function toAxis(source, offset) {
  *
  * Ref: https://github.com/immersive-web/cardboard-vr-display/pull/19
  */
-export const getChromeVersion = (() => {
-	const match = userAgent.match(/.*Chrome\/([0-9]+)/);
-	const value = match ? parseInt(match[1], 10) : -1;
-
-	return () => value;
-})();
-
 /**
  * In Chrome m65, `devicemotion` events are broken but subsequently fixed
  * in 65.0.3325.148. Since many browsers use Chromium, ensure that
  * we scope this detection by branch and build numbers to provide
  * a proper fallback.
  * https://github.com/immersive-web/webvr-polyfill/issues/307
  */
-export const isChromeWithoutDeviceMotion = (() => {
-	let value = false;
-
-	if (getChromeVersion() === 65) {
-		const match = userAgent.match(/.*Chrome\/([0-9.]*)/);
+let version = -1; // It should not be null because it will be compared with number
+let branch = null;
+let build = null;
 
-		if (match) {
-			const versionToken = match[1].split(".");
-			const branch = versionToken[2];
-			const build = versionToken[3];
+const match = /Chrome\/([0-9]+)\.(?:[0-9]*)\.([0-9]*)\.([0-9]*)/i.exec(userAgent);
 
-			value = parseInt(branch, 10) === 3325 && parseInt(build, 10) < 148;
-		}
-	}
-	return () => value;
-})();
+if (match) {
+	version = parseInt(match[1], 10);
+	branch = match[2];
+	build = match[3];
+}
 
-export const isAndroid = () => /Android/i.test(userAgent);
```

Comment:
@jongmoon 

I think you should be at consts.js rather than utils.js.

Reply:
@younkue you are right!